setup.py

```diff
@@ -68,8 +68,8 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'molecule==1.25.1',
-        'ansible==2.3.2',
-        'docker',
+        'ansible==2.6.3',
+        'docker-compose',
```

Comment:
Shouldn't this be `docker`?

Reply:
I copied this requirement from https://github.com/openmicroscopy/ome-ansible-molecule-dependencies/pull/10/files#diff-2eeaed663bd0d25b7e608891384b7298R74 it pulls `docker` as a dependency

```
Collecting docker<4.0,>=3.4.1 (from docker-compose->ome-ansible-molecule-dependencies==0.3.2.dev1)
```

Reply:
OK, can you make this a versioned dependency, then this should be good to merge

Reply:
Pushed. 16e4250 also contains the discussed renaming of this meta-package using the proposed `ome-ansible-molecule` suggestion for now although there might be better/shorted/more concise names clarifying the scope of this meta-package.

Once we have the name settled, the repo will need to be renamed before cutting a `0.4.0` release to PyPI and the new package can be consumed across all Ansible roles.

Reply:
Thanks, I'll give it a final test with https://github.com/IDR/deployment/pull/135 before merging. `ome-ansible-molecule` sounds fine to me, not over long but not too generic.